.github/workflows/.tilbakestill.yaml

```diff
@@ -0,0 +1,25 @@
+name: .tilbakestill.yaml
```

Comment:
Kanskje vi kan kalle denne deploy-main? Tilbakestill kan bety flere ting for meg 😄